src/components/GameArea/OptionItem.tsx

```diff
@@ -0,0 +1,73 @@
+import { useGameContext } from "../../context/game-context";
+import { OptionItemProps } from "../../pages/GameArea/GameArea.types";
+
+export const OptionItem = ({
+  isRight,
+  isSelected,
+  text,
+  index,
+  currentQuestionIndex,
+  dispatch,
+  swipeToNextQuestion,
+}: OptionItemProps): JSX.Element => {
+  const {
+    dataState: { questions },
+    dataDispatch,
+  } = useGameContext();
+
+  const setOptionStyle = (
+    isOptionSelected: Boolean,
+    isOptionRight: Boolean
+  ): string => {
+    if (isOptionSelected && isOptionRight) {
+      return "option-btn option-correct";
+    } else if (isOptionSelected && !isOptionRight) {
```

Comment:
The `else` is not needed here, just use multiple `if` statements as you are already doing early return.